.github/workflows/go-test.yml

```diff
@@ -0,0 +1,25 @@
+on: [push]
+name: Golang Tests
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.16.x]
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Grab modules
+        run: go get ./...
```

Comment:
I don't think this step is necessary since installing the modules is automatically executed by `make test (go test)`.

Reply:
953853dc